open-metadata-implementation/view-services/pom.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project. -->
```

Comment:
We need to add a new section to all poms

See the <scm> section which is now in all of them.

Reply:
`<scm>`
(That tag got removed in my comment!)